plugins/discourse-local-dates/test/javascripts/acceptance/local-dates-composer-test.js

```diff
@@ -128,4 +131,26 @@ acceptance("Local Dates - composer", function (needs) {
     await click("ul.formats a.moment-format");
     assert.dom("input.format-input").hasValue("LLL");
   });
+
+  test("composer insert current time shortcut", async function (assert) {
+    await visit("/t/internationalization-localization/280");
+
+    await click("#topic-footer-buttons .btn.create");
+    assert.dom(".d-editor-input").exists("the composer input is visible");
+    await fillIn(".d-editor-input", "and the time now is: ");
+
+    const date = moment().format("YYYY-MM-DD");
+
+    await triggerKeyEvent(".d-editor-input", "keydown", ".", {
+      ...metaModifier,
+      shiftKey: true,
+    });
+
+    assert
+      .dom("#reply-control .d-editor-input")
+      .hasValue(
+        new RegExp(`and the time now is: \\[date=${date}`),
+        "it adds the current date"
+      );
+  });
```

Comment:
Super nitpick, but I like to make assertions standout in a test:

```suggestion
  test("composer insert current time shortcut", async function (assert) {
    await visit("/t/internationalization-localization/280");
    await click("#topic-footer-buttons .btn.create");
    
    assert.dom(".d-editor-input").exists("the composer input is visible");
    
    await fillIn(".d-editor-input", "and the time now is: ");
    const date = moment().format("YYYY-MM-DD");
    await triggerKeyEvent(".d-editor-input", "keydown", ".", {
      ...metaModifier,
      shiftKey: true,
    });

    assert
      .dom("#reply-control .d-editor-input")
      .hasValue(
        new RegExp(`and the time now is: \\[date=${date}`),
        "it adds the current date"
      );
  });
```